src/shared/components/post/post-listing.tsx

```diff
@@ -345,7 +345,7 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
           )}
         </li>
         {post_view.post.language_id !== 0 && (
-          <span className="mx-1 badge badge-muted">
+          <span className="mx-1 badge">
```

Comment:
You're not supposed to use it without the 2nd `badge-x` class, as it might look weird on some themes. Did you try `badge-secondary` ?

Reply:
Secondary will be orange on lightly I think. It'll bring us back to the badge being too prominent.

Reply:
Well its working...

Reply:
If it looks good on both the litely and darkly, I have no problems merging. Might give issues on other themes tho.